externals/CMakeLists.txt

```diff
@@ -138,6 +138,11 @@ endif()
 # MicroProfile
 add_library(microprofile INTERFACE)
 target_include_directories(microprofile SYSTEM INTERFACE ./microprofile)
+if (ENABLE_MICROPROFILE)
+    target_compile_definitions(microprofile INTERFACE MICROPROFILE_ENABLED=1)
```

Comment:
For the sake of simplicity I would suggest changing `MICROPROFILE_ENABLED` to also be named `ENABLE_MICROPROFILE`

Reply:
Same as before, I don't want to change the code in an external. I used `ENABLE_MICROPROFILE` to be consistent with other options.

Reply:
Oh yeah, I see now that this is a definition from microprofile itself. In that case, this is fine.